runtime/src/common/sgx/avr.rs

```diff
@@ -316,14 +298,36 @@ pub fn verify(avr: &AVR) -> Fallible<AuthenticatedAVR> {
         _ => return Err(AVRError::MalformedQuote.into()),
     };
 
+    if option_env!("OASIS_UNSAFE_ALLOW_DEBUG_ENCLAVES").is_none() {
+        // Drop debug enclaves, if we are in production mode.
+        if quote_body
```

Comment:
This could be simplified by something like:
```rust
let is_debug = quote_body.report_body...contains(AttributeFlags::DEBUG);
let allow_debug = option_env!("...").is_none();
if is_debug != allow_debug { /* error */ }
```